c++/src/linked-list.cc

```diff
@@ -101,6 +108,8 @@ class LinkedList {
    * @param int   the value to be appeneded
    */
   void append(T value) {
+    if (this->head == nullptr) {this->head = new Node<T>(value);return;} 
```

Comment:
I've seen you do this in previous commits. Don't do this. Nothing is saved by cramming everything on one line without spacing. This should be:

``` cpp
if (this->head == nullptr) {
  this->head = new Node<T>(value);
  return;
}
```